config/helm/appmesh-controller/values.yaml

```diff
@@ -8,6 +8,8 @@ accountId: ""
 preview: false
 enableBackendGroups: false
 clusterName: ""
+useAwsDualStackEndpoint: 0
+useAwsFIPSEndpoint: 0
```

Comment:
These are essentially boolean flags, right? 

I believe the convention here is to keep them as boolean, then convert them to "0" / "1" specifically when needed if being converted to environment variables (same as you did in the prior FIPS patch).

It so happens that in this case, we need to situationally convert true/false values to 1/0. It's best to preserve the "true structure" of the boolean flags throughout the code so the meaning is not lost, then convert at the edges depending on what external systems expect.